README.md

```diff
@@ -56,6 +56,17 @@ onnxslim your_onnx_model slimmed_onnx_model
 
 For more usage, see onnxslim -h or refer to our [examples](./examples)
 
+# Used by OnnxSlim
```

Comment:
I think using "Projects using OnnxSlim" is more accurate

Reply:
> I think using "Projects using OnnxSlim" is more accurate

This is a good suggestion. done.